backend/src/main/java/com/redhat/cloud/notifications/models/InternalRoleAccess.java

```diff
@@ -31,8 +35,14 @@
     private String role;
 
     @NotNull
+    @Transient
     private UUID applicationId;
 
+    @ManyToOne(fetch = FetchType.EAGER)
```

Comment:
`@ManyToOne` is `EAGER` by default, but don't push another commit just for that 😄